bower.json

```diff
@@ -3,12 +3,13 @@
   "homepage": "https://github.com/garyb/purescript-strongcheck-laws",
   "description": "Law tests for core classes",
   "keywords": [
-    "purescript"
+    "purescript",
+    "pursuit-deprecated"
```

Comment:
The `pursuit-deprecated` keyword signals to Pursuit that the package should be marked as deprecated.